### PR TITLE
Replace pear.phpunit.de with composer and phar.phpunit.de

### PR DESCRIPTION
**Pull request by styks1987**
As of Dec. 31st 2014, phpunit is no longer available as a pear package.

### CODE REVIEW

File: en/development/testing.rst [hunk 1]
```diff
@@ -12,13 +12,25 @@ Installing PHPUnit
 
 CakePHP uses PHPUnit as its underlying test framework. PHPUnit is the de-facto
 standard for unit testing in PHP. It offers a deep and powerful set of features
-for making sure your code does what you think it does. PHPUnit can be installed
-through the `pear installer <http://pear.php.net>`_. To install PHPUnit run the
-following::
+for making sure your code does what you think it does.::
+
+As of Dec. 31st, 2014 installation via pear.phpunit.de is no longer valid.
```
Comment (markstory) [inline en/development/testing.rst]: I don't think this is necessary, as time travellers from the past are pretty infrequent :smile:

Comment (styks1987) [inline en/development/testing.rst]: Maybe it is just too late, but I don't quite understand what your comment means. :) My guess is that it is just extraneous information that I need to remove.

Comment (markstory) [inline en/development/testing.rst]: I was trying to say that with the date in the past, we don't need to mention the date.

**Comment by markstory** [comment]
Thanks :+1:

**Pull request merged**